tools/pythonpkg/duckdb-stubs/__init__.pyi

```diff
@@ -46,7 +46,7 @@ class DuckDBPyConnection:
     def close(self) -> None: ...
     def commit(self) -> DuckDBPyConnection: ...
     def cursor(self) -> DuckDBPyConnection: ...
-    def df(self, df: pandas.DataFrame) -> DuckDBPyRelation: ...
+    def df(self) -> Any: ...
```

Comment:
Should this not return a `pandas.DataFrame`, instead of `Any`?

Reply:
I was also thinking that, but `to_df` has the same signature iirc
I figured we probably do it this way because we don't have a direct dependency on Pandas?

Reply:
We work around this with a false import and the `DataFrame`class in the pybind11 binding - we should probably fix this now